### PR TITLE
Clear all filters button

### PR DESCRIPTION
**Pull request by sivang**
Creates a 'Clear All' button on the filters facet to clear all filters set before
Addresses JIRA: https://open-data-ed.atlassian.net/browse/OD-221

### CODE REVIEW

File: ckanext/ed/templates/package/search.html [hunk 1]
```diff
@@ -22,6 +22,8 @@
                     {{ h.snippet('snippets/filter_list.html', title=c.facet_titles[facet], name=facet) }}
                     {% endif %}
             {% endfor %}
+            <button onclick="javascript: window.location.href = '{{ h.url_for('dataset') }}'">Clear All</button>
```
Comment (zelima) [inline ckanext/ed/templates/package/search.html]: @sivang this is simpler, no? We could avoid javascript
```
<a class="usa-button" href="{{ h.url_for('dataset') }}">Clear All</a>
```

Comment (zelima) [reply ckanext/ed/templates/package/search.html]: cc @tanvirchahal which one is better to style?

Comment (sivang) [reply ckanext/ed/templates/package/search.html]: Yeah,  then we do not need the javascript, but @tanvirchahal - is it mandatory to have a button as the HTML element or could we get away with an `a href` ?

Comment (zelima) [reply ckanext/ed/templates/package/search.html]: @sivang in most cases we actually  have `<a href=` so go ahead and merging

Comment (sivang) [reply ckanext/ed/templates/package/search.html]: as per @tanvirchahal it doesn't matter what element we use, so changing to a href to drop usage of javascript

**Pull request merged**